src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise(resolve => {
+  logo.addEventListener('click', resolve);
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(Error);
```

Comment:
You should pass an error message to the Error constructor, like `Error('Promise rejected')` or invoke your errorMessage